packages/event-listener/package.json

```diff
@@ -23,7 +23,7 @@
         "test": "rm -rf db.json && npm run build && mocha dist/js/test/ --timeout 60000 --exit",
         "test:concurrent": "concurrently --success first --kill-others -n eth,backend,test \"yarn start-ganache\" \"yarn start-test-backend\" \"yarn test\"",
         "start-test-backend": "node ../../node_modules/@energyweb/utils-testbackend/dist/js/src/index.js",
-        "start-ganache": "ganache-cli -m 'chalk park staff buzz chair purchase wise oak receive avoid avoid home' -l 8000000 -e 1000000 -a 20",
+        "start-ganache": "ganache-cli -m 'chalk park staff buzz chair purchase wise oak receive avoid avoid home' -l 8000000 -e 1000000 -a 20 -b 0,5",
```

Comment:
Is `-b` flag required?

It's discouraged by Ganache CLI:
```
Using the --blockTime flag is discouraged unless you have tests which require a specific mining interval.
```

Reply:
It is needed in this case to make the tests work, otherwise it doesn't handle events properly (if there is an event in the last block, it will keep sending notifications until there is a new block mined - which is never).

Reply:
Will work on a better solution once the features are done.